.github/pull_request_template.md

```diff
@@ -7,9 +7,6 @@ Closes #[issue number]
 
 **Special notes for your reviewer**:
 
-**How does this PR make you feel**:
```

Comment:
I kind of liked this in the old template! Maybe not the giphy link.

Reply:
Yeah, I thought it was pretty cute but maybe it's a bit silly and unnecessary... I'm cool either way.

Reply:
I'm a curmudgeon and removed it but I can put it back if folks liked it =P

Reply:
I kinda liked it, but not strongly motivated either way.